release.yml

```diff
@@ -0,0 +1,70 @@
+stages:
+- stage: Release
+  jobs:
+  - job: Release
+    pool:
+      vmImage: 'Ubuntu-16.04'
+    continueOnError: true
+    steps:
+    - task: AzureKeyVault@1
+      displayName: 'Azure Key Vault: ado-secrets'
+      inputs:
+        azureSubscription: 'ClientToolsInfra_670062 (88d5392f-a34f-4769-b405-f597fc533613)'
+        KeyVaultName: 'ado-secrets'
+        SecretsFilter: 'github-distro-mixin-password,ado-crossplatbuildscripts-password'
+    - task: PowerShell@2
```

Comment:
I understand this is just the yml that is created by the ui, I prefer the simplier syntax for this.
````
- powershell: |
    SCRIPT GOES HERE
  displayname: ````